src/simple.ts

```diff
@@ -0,0 +1,42 @@
+import { defaultAbiCoder } from '@ethersproject/abi';
+import { BytesLike, HexString, toHex } from './bytes';
+import { MultiProof, processProof, processMultiProof } from './core';
+import { MerkleTreeData, MerkleTreeImpl } from './merkletree';
+import { MerkleTreeOptions } from './options';
+import { validateArgument } from './utils/errors';
+
+export type SimpleMerkleTreeData = MerkleTreeData<BytesLike> & {
```

Comment:
I think we will want this to always be `HexString`. For example, arrays of numbers should not be valid in a dump.

```suggestion
export type SimpleMerkleTreeData = MerkleTreeData<HexString> & {
```

Reply:
That works for the `SimpleMerkleTree`, but for the `StandardMerkleTree`, `T extends any[]` may contain crazy things that will end up in dumps.